tsdb/chunks/chunks.go

```diff
@@ -311,6 +316,48 @@ func MergeChunks(a, b chunkenc.Chunk) (*chunkenc.XORChunk, error) {
 	return newChunk, nil
 }
 
+// BreakLargeChunks splits a chunk into many if it has more than a certain number of samples.
+func BreakLargeChunks(chks []Meta, limit int) ([]Meta, error) {
```

Comment:
Does it have to be public?

Reply:
No it doesn't. I'll change it. Thanks for the review.